README.md

```diff
@@ -0,0 +1,9 @@
+# shar
+Yet another tool for video streaming (WIP)
+## Installing
+1. Git clone
```

Comment:
I think better to write build instruction in "Command line style" (feel free to use this readme as example https://github.com/pl0q1n/RustyHGI/blame/master/README.md#L17)